src/index.ts

```diff
@@ -1,130 +0,0 @@
-import {configLogs, error, getErrorMessage, info} from "./utils/logger";
-import {Config, loadConfig} from "./utils/configFileReader";
-import {PullRequestInfo} from "./pullRequestInfo";
-import {PipelineRun} from "./utils/piplineRun";
-import {Deployment, KubernetesAPI} from "./utils/kubernetes";
-
-const args = process.argv.slice(2);
-const configFileIndex = args.findIndex(arg => arg === '-p');
-let config: Config | undefined;
-
-const pjson = require('../package.json');
-const version = pjson.version;
-
-if (configFileIndex !== -1 && configFileIndex + 1 < args.length) {
-	const configFilePath = args[configFileIndex + 1];
-	info(`Using config file path: ${configFilePath}`, '');
-
-	config = loadConfig(configFilePath);
-	if (!config) {
-		throw new Error("Error loading config file")
-	}
-	configLogs(config);
-} else {
-	throw new Error("Config file not specified")
-}
-
-info(`wp-continuous-integration started with version ${version}`, {});
-
-async function scheduleActivePRsToDeployments() {
-	const pullRequests = await PullRequestInfo.getAvailablePRsSortedByDate(config!);
```

Comment:
(Here and elsewhere) please replace `config!` with `config`, as per the comment above